src/current/v23.2/column-level-encryption.md

```diff
@@ -0,0 +1,331 @@
+---
+title: Column Level Encryption
+summary: SQL users are required to pass encryption keys when reading from or writing to specific columns
+toc: true
+docs_area: reference.sql
+---
+
+CockroachDB v23.2 and later support _column level encryption_. Using this feature, you can encrypt one or more of the columns in each row of a database table.
+
+Column level encryption can be useful for compliance scenarios such as adhering to PCI or GDPR.
```

Comment:
I don't know about this being useful for GDPR. So I think we should just stick to PCI.

Reply:
sounds good, updated to end the sentence at "... adhering to PCI."